tests/Maker/MakeEntityTest.php

```diff
@@ -566,5 +566,181 @@ public function getTestDetails()
             ->configureDatabase()
             ->updateSchemaAfterCommand(),
         ];
+
```

Comment:
I need to add more tests. Maybe a functionnal test in a new fixtures (maybe MakeApiResource) wich request the api with correspondig filters. Also we probably need to test the cases where the filters are not supposed to match the type assigned to the current field. I have another problem, the TermFilter and MatchFilter filters need Elasticsearch